readme.md

```diff
@@ -48,35 +55,45 @@ Life Cycle of a request/response in Rails:
 
 4. Once the request hits the controller, its then going to query the database through Active Record(the model) for the information specified in the controller.
 
-5. Once the controller has the information from the model that it needs it sends it to the view
+5. Once the controller has the information from the model that it needs, it sends it to the view
+
+6. The view takes the objects from the controller, and sends a response to the user.
```

Comment:
LOTS of information here, so they're probably going to be all "woah" and ask lots of questions. Make sure you timebox questions and let them know you're just going over the "big picture" right now. Familiarity with it will come with time.